Design/proctoring page sammy

Sammy-prog-cloud opened the pull request:
Added user login section with secure login area for the Online Proctoring Exams.

Sammy-prog-cloud committed:
Add files via upload

"Added user login section with secure login area"

Sammy-prog-cloud committed:
Added user login section (Design2)

Another design

Sammy-prog-cloud committed:
Delete Online Proctoring Exam (Desktop and phone  login).fig

Sammy-prog-cloud committed:
Delete Online Proctoring Login Design1.fig

Sammy-prog-cloud committed:
Added user login section with secure login area

Sammy-prog-cloud committed:
Delete Online Proctoring Login directory

Deleted Login page

Sammy-prog-cloud committed:
Added user login section with secure login area 

Changed the color and added screenshot of it.

Sammy-prog-cloud committed:
Added user login section with secure login area 

Contains the screenshot of the login section and the design of the login section

Sammy-prog-cloud committed:
Delete Online Proctoring Login directory

Contained some rough work that are  not needed.

yimjohns commented:
Looks good!

yimjohns reviewed (approved):
Looks Good!

ayodeji reviewed (approved):
Looks very nice thanks

Sammy-prog-cloud committed:
Added instruction Area with clear instructions 

Concise and clear instructions, how to set up the camera and how their environment should be.

Sammy-prog-cloud committed:
Added live camera feed with option to minimize

Option to switch to video before starting the exam is available in the design
